lib/adapters/QUnitAdapter.js

```diff
@@ -58,27 +111,21 @@ export default class QUnitAdapter extends EventEmitter {
   }
 
   onTestDone (details) {
-    var test = this.idDataMap[details.testId]
-
-    if (details.failed !== 0) {
-      test.status = 'failed'
-    } else if (details.skipped === true) {
-      test.status = 'skipped'
-    } else {
-      test.status = 'passed'
-    }
-
-    test.runtime = details.runtime
-    test.errors = this.errors
-
-    this.emit('testEnd', test)
+    details.testEnd = true
```

Comment:
Same as above. Your comment doesn't reflect the implementation. And I don't think we should be adding even more properties to the data we're consuming. We really should consider those objects immutable and track these properties ourselves, like we now do with `this.testsDetails`.